angr/procedures/libc/feof.py

```diff
@@ -15,5 +15,5 @@ def run(self, file_ptr):
         fileno = self.state.mem[file_ptr + fd_offset:].int.concrete
         simfd = self.state.posix.get_fd(fileno)
         if simfd is None:
-            return None
+            return self.state.solver.If('feof', self.state.solver.BVV(1, self.state.arch.bits), 0)
```

Comment:
Why this is an `If` here. The string should always evaluate to True.

Reply:
oh that‘s a mistake as it should create a new symbolic variable instead of string

Reply:
If one pass NULL to feof should it not crash instead?

Reply:
changed in latest commit